sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregator/AggregatorFactory.java

```diff
@@ -5,16 +5,42 @@
 
 package io.opentelemetry.sdk.metrics.aggregator;
 
+import io.opentelemetry.sdk.metrics.aggregation.Accumulation;
 import javax.annotation.concurrent.Immutable;
 
 /** Factory class for {@link Aggregator}. */
 @Immutable
-public interface AggregatorFactory {
+public abstract class AggregatorFactory {
 
   /**
-   * Returns a new {@link Aggregator}.
+   * Returns a new {@link Aggregator}. This MUST by used by the synchronous to aggregate recorded
+   * measurements during the collection cycle.
    *
    * @return a new {@link Aggregator}.
    */
-  Aggregator getAggregator();
+  public abstract Aggregator getAggregator();
+
+  /**
+   * Returns a new {@code Accumulation} for the given value. This MUST be used by the asynchronous
+   * instruments to create {@code Accumulation} that are passed to the processor.
+   *
+   * @param value the given value to be used to create the {@code Accumulation}.
+   * @return a new {@code Accumulation} for the given value.
+   */
+  public Accumulation accumulateLong(long value) {
```

Comment:
discussed offline that we should probably move these methods into the Aggregator. Not sure if this should be done in this PR or a follow-up.

Also, it might be nice to call it something like "directAccumulate" or something that makes it clear it should be used by async instruments.

Reply:
If you are ok I would do it in the next PR :)